### ISSUE TITLE
Is it possible to use these snippets in org-mode?

### ISSUE THREAD

**Issue by maikol-solis** [tecosaur/LaTeX-auto-activating-snippets#4]
I use org-mode to write all my works. I write 99% of the time latex formulas, equations, etc. 

Is it any chance that the `auto-activating-snippets` work in org-mode?

Best.

**Comment by K-gihu** [comment]
@maikol-solis 
This package uses `aas-set-snippets`(defined in `auto-activating-snippets.el`) to set available snippets for a particular major-mode.
Specifically, at [here](https://github.com/tecosaur/LaTeX-auto-activating-snippets/blob/262d4035d36dfba7a58510bdab6aad9c4ae8c4df/latex-auto-activating-snippets.el#L351), the package set its snippets for latex-mode to use.
```emacs-lisp
(apply #'aas-set-snippets 'latex-mode laas-basic-snippets)
(apply #'aas-set-snippets 'latex-mode laas-subscript-snippets)
(apply #'aas-set-snippets 'latex-mode laas-frac-snippet)
(apply #'aas-set-snippets 'latex-mode laas-accent-snippets)
```

So I just set the four snippets for org-mode to use. Here is my doom configuration for reference.
```emacs-lisp
(use-package! auto-activating-snippets
  :hook (org-mode . auto-activating-snippets-mode)
  :config ; do whatever here
  (use-package! latex-auto-activating-snippets)
  (apply #'aas-set-snippets 'org-mode laas-basic-snippets)
  (apply #'aas-set-snippets 'org-mode laas-subscript-snippets)
  (apply #'aas-set-snippets 'org-mode laas-frac-snippet)
  (apply #'aas-set-snippets 'org-mode laas-accent-snippets)
  )
```

**Comment by maikol-solis** [comment]
Thank you very much @K-gihu. Perfect answer!

**Comment by ymarco** [comment]
FYI after discussion in the package's application to MELPA, the snippets were moved to their own minor mode `laas-mode`.
So on the current commit it should just take something like
```emacs-lisp
(add-hook 'org-mode #'laas-mode)
```
And `laas-mode` takes care of turning on `aas-mode` as well.
For actual full org-mode support, see #5.

Sorry for being unresponsive, I only now remembered to turn on notifications here.
